src/find/matchers/perm.rs

```diff
@@ -0,0 +1,480 @@
+// Copyright 2017 Google Inc.
+//
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+/// ! find's permission matching uses a very unix-centric approach, that would
```

Comment:
First, I'm unfamiliar with the '!' syntax, did you mean "//!" ? Second, I get a feeling the comment is disconnected from something, maybe a #[cfg] attribute.